code/game/gamemodes/cult/runes.dm

```diff
@@ -377,25 +377,24 @@ var/list/teleport_other_runes = list()
 /obj/effect/rune/sacrifice/proc/sac(mob/living/T)
 	var/sacrifice_fulfilled
 	if(T)
-		if(istype(T, /mob/living/simple_animal/pet/dog/corgi))
-			for(var/mob/living/carbon/C in orange(1,src))
+		if(istype(T, /mob/living/simple_animal/pet))
+			for(var/mob/living/carbon/C in range(3,src))
 				if(iscultist(C))
-					C << "<span class='warning'>Even dark gods from another plane have standards, sicko.</span>"

```

Comment:
Um no, it wasn't stupid, this is one of my favorite things in game, and I can't stand for you to remove it.  If this is removed, I'll likely make a PR adding it back.